lib/include/dlisio/lis/protocol.hpp

```diff
@@ -11,6 +11,16 @@
 
 namespace dlisio { namespace lis79 {
 
+/** Size of LIS type
+ *
+ * Return the size of bytes needed to store the LIS type. E.g. lis::f32 is a 4
+ * byte type. Variable length types such as lis::string and lis::mask return 0.
```

Comment:
I would prefer exception or -1?
0 seems like a valid value. Data not weights anything? Well, ok?
-1 gives clear indication something is wrong.